components/kyma-environment-broker/internal/provider/gcp_provider.go

```diff
@@ -21,6 +25,7 @@ func (p *GcpInput) Defaults() *gqlschema.ClusterConfigInput {
 			MachineType:    "n1-standard-4",
 			Region:         DefaultGCPRegion,
 			Provider:       "gcp",
+			Purpose:        &DefaultGCPPurpose,
```

Comment:
do not specify the purpose here, the line is not needed

Reply:
Done!